packages/coinstac-api-server/src/database/migrations/scripts/2022-03-28-add-active-members-to-consortium.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+const database = require('../../index');
+
+const description = 'Add active members property to consortia';
+
+async function up() {
+  const db = database.getDbInstance();
+
+  db.collection('consortia').updateMany({}, [
+    { $set: { activeMembers: "$members" } }
+  ]);
+}
+
+async function down() {
+  const db = database.getDbInstance();
+
+  // Downgrade code goes here
```

Comment:
shouldnt this unset the obj?

Reply:
yeah... It's just that we don't have any use for downgrade right now, but I'll add the code for it.

Reply:
nbd